simulators/ethereum/eth/Dockerfile

```diff
@@ -0,0 +1,20 @@
+# This simulation runs various graphql tests
```

Comment:
Does it?

Reply:
No it does not :) you caught me copy pasting.